tools/cross/format.py

```diff
@@ -0,0 +1,135 @@
+import logging
```

Comment:
Nit: add something like `#!/usr/bin/env python3` here and add execute permissions to the file to allow running it as just `./tools/cross/format.py` (i.e. without explicitly listing python in the command)

Reply:
I added the `#!usr/bin/env python3` but I don't have permission to add execute permissions to files.

Reply:
> I added the `#!usr/bin/env python3` but I don't have permission to add execute permissions to files.

You don't have permission as in it violates development guidelines for workerd or as in the system doesn't let you? Does `chmod +x tools/cross/format.py` not work?

Reply:
Chmod works, when I try to push I get the error.

Reply:
Ok that's weird, we certainly have scripts with execute permissions in the repo, e.g. `tools/unix/workspace-status.sh`. I think this is non-blocking overall but what error message do you get?